From d4e9686477c629f37080153d2d358c0a8808d8f7 Mon Sep 17 00:00:00 2001
From: Ishimoto Shinobu <nagakamira@gmail.com>
Date: Sun, 14 Feb 2021 18:44:38 +0900
Subject: [PATCH 4/4] openssl-src: add more linux-musl targets

Signed-off-by: Ishimoto Shinobu <nagakamira@gmail.com>
---
 vendor/openssl-src/src/lib.rs | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/vendor/openssl-src/src/lib.rs b/vendor/openssl-src/src/lib.rs
index 4019e8123..1a0c88c4a 100644
--- a/vendor/openssl-src/src/lib.rs
+++ b/vendor/openssl-src/src/lib.rs
@@ -191,7 +191,9 @@ impl Build {
             "mips-unknown-linux-gnu" => "linux-mips32",
             "mips-unknown-linux-musl" => "linux-mips32",
             "mips64-unknown-linux-gnuabi64" => "linux64-mips64",
+            "mips64-unknown-linux-muslabi64" => "linux64-mips64",
             "mips64el-unknown-linux-gnuabi64" => "linux64-mips64",
+            "mips64el-unknown-linux-muslabi64" => "linux64-mips64",
             "mipsel-unknown-linux-gnu" => "linux-mips32",
             "mipsel-unknown-linux-musl" => "linux-mips32",
             "powerpc-unknown-linux-gnu" => "linux-ppc",
-- 
2.29.2

