From c92db3141f28812852109fc3a4ba174116247b93 Mon Sep 17 00:00:00 2001
From: Ishimoto Shinobu <nagakamira@gmail.com>
Date: Wed, 17 Feb 2021 06:53:42 +0900
Subject: [PATCH 4/5] openssl-src: add more linux-musl targets

Signed-off-by: Ishimoto Shinobu <nagakamira@gmail.com>
---
 vendor/openssl-src/src/lib.rs | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/vendor/openssl-src/src/lib.rs b/vendor/openssl-src/src/lib.rs
index 4019e8123..b55564dbf 100644
--- a/vendor/openssl-src/src/lib.rs
+++ b/vendor/openssl-src/src/lib.rs
@@ -191,15 +191,21 @@ impl Build {
             "mips-unknown-linux-gnu" => "linux-mips32",
             "mips-unknown-linux-musl" => "linux-mips32",
             "mips64-unknown-linux-gnuabi64" => "linux64-mips64",
+            "mips64-unknown-linux-muslabi64" => "linux64-mips64",
             "mips64el-unknown-linux-gnuabi64" => "linux64-mips64",
+            "mips64el-unknown-linux-muslabi64" => "linux64-mips64",
             "mipsel-unknown-linux-gnu" => "linux-mips32",
             "mipsel-unknown-linux-musl" => "linux-mips32",
             "powerpc-unknown-linux-gnu" => "linux-ppc",
+            "powerpc-unknown-linux-musl" => "linux-ppc",
             "powerpc64-unknown-freebsd" => "BSD-generic64",
             "powerpc64-unknown-linux-gnu" => "linux-ppc64",
+            "powerpc64-unknown-linux-musl" => "linux-ppc64",
             "powerpc64le-unknown-freebsd" => "BSD-generic64",
             "powerpc64le-unknown-linux-gnu" => "linux-ppc64le",
+            "powerpc64le-unknown-linux-musl" => "linux-ppc64le",
             "riscv64gc-unknown-linux-gnu" => "linux-generic64",
+            "riscv64gc-unknown-linux-musl" => "linux-generic64",
             "s390x-unknown-linux-gnu" => "linux64-s390x",
             "x86_64-apple-darwin" => "darwin64-x86_64-cc",
             "x86_64-linux-android" => "linux-x86_64",
-- 
2.29.2

